src/main/java/executor/api/controller/ScenarioSourceController.java

```diff
@@ -0,0 +1,29 @@
+package executor.api.controller;
+
+import executor.api.model.ScenarioDTO;
+import executor.api.service.ScenarioSourceQueueHandlerImpl;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/scenario")
```

Comment:
let's add /api/v1/ prefix for all api endpoints
```suggestion
@RequestMapping("/api/v1/scenario")
```